### PR TITLE
add otp

### PR DESCRIPTION
**Pull request by sahildesai313**
# Description
- Added Forget password.

### CODE REVIEW

File: LR/LR/settings.py [hunk 1]
```diff
@@ -124,3 +124,45 @@
 # https://docs.djangoproject.com/en/5.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+
+
+
+
+
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'  # Or your preferred backend
```
Comment (tarang1810) [inline LR/LR/settings.py:133]: Put these secret credentials in a .env file and import from it.

File: LR/LR/settings.py [hunk 2]
```diff
@@ -124,3 +124,45 @@
...
+EMAIL_USE_TLS = True
+EMAIL_HOST_USER = 'sahilpatel313200@gmail.com'
+EMAIL_HOST_PASSWORD = 'kapi cous pffd fvsx'
+DEFAULT_FROM_EMAIL = 'sahilpatel313200@gmail.com' 
+
+
+
+
+ 
+    # def post(self,request):
```
Comment (tarang1810) [inline LR/LR/settings.py:145]: If this code is not worked then remove it.

File: LR/glasses/serializers.py [hunk 3]
```diff
@@ -19,3 +22,17 @@ def create(self, validated_data):
...
 
+
+class forgetserializer(serializers.ModelSerializer):
+    otp=serializers.IntegerField()
+
+    
+    class Meta:
+        model = person 
+        fields = ['email','password','confirmpassword','otp']
+
```
Comment (tarang1810) [inline LR/glasses/serializers.py:33]: Remove extra lines 
Only put one line at the end of the page.

File: LR/glasses/views.py [hunk 4]
```diff
@@ -68,32 +72,88 @@ def post(self, request):
...
+            password = request.POST.get("password")
+            confirmpassword = request.POST.get("confirm_Password")
+
+            if password == confirmpassword:
+                User = person.objects.all
+                new_password = password
+                User.password = new_password
+                User.save()
+                return redirect("login")
+
```
Comment (tarang1810) [inline LR/glasses/views.py:146]: Please push working code.

**Review by tarang1810 — CHANGES_REQUESTED** [review]

File: LR/LR/settings.py (hunk 2 above)
Comment (sahildesai313) [reply LR/LR/settings.py:145]: okay

File: LR/glasses/serializers.py (hunk 3 above)
Comment (sahildesai313) [reply LR/glasses/serializers.py:33]: ok

File: LR/glasses/views.py (hunk 4 above)
Comment (sahildesai313) [reply LR/glasses/views.py:146]: ok I check before pushing  it

**Review by nikunj482 — APPROVED** [review]

**Review by tarang1810 — APPROVED** [review]

**Pull request merged**